create-vite-express/templates/vanilla-ts/tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "useDefineForClassFields": true,
     "module": "CommonJS",
     "lib": ["ESNext", "DOM"],
-    "moduleResolution": "Node",
+    "moduleResolution": "Node10",
```

Comment:
I see that `Node` is now deprecated and called `Node10`, so thank you for changing that, but maybe we should use `node16` or even `nodenext` as it is the [default](https://www.typescriptlang.org/tsconfig#moduleResolution) now for modern versions?

Reply:
Ah yes, thanks for showing me  this. I'll update the  templates to use the default strategy.